raft/src/test/java/org/apache/kafka/raft/LeaderStateTest.java

```diff
@@ -866,6 +872,27 @@ public void testGrantVote(boolean isLogUpToDate) {
         assertFalse(state.canGrantVote(ReplicaKey.of(3, Optional.empty()), isLogUpToDate));
     }
 
+    @Test
+    public void testBeginQuorumEpochTimer() {
+        int leader = localId;
+        int follower1 = 1;
+        long epochStartOffset = 10L;
+
+        Set<Integer> voterSet = mkSet(leader, follower1);
+        LeaderState<?> state = newLeaderState(voterSet, epochStartOffset);
+        assertEquals(0, state.timeUntilBeginQuorumEpochTimerExpires(time.milliseconds()));
+
+        long resetTime = time.milliseconds();
```

Comment:
`time.milliseconds` is mocked so it will return the same value as long as you don't call `time.sleep(long)`.

Reply:
I can add a sleep before this, the main point was just to test the reset functionality though.

Reply:
It is unnecessary to save the value in a variable.

Reply:
I'll remove it - just thought it would make it extremely clear the times are supposed to be equal for that check